src/tools/dev/xml/GenerateCMake.h

```diff
@@ -281,30 +281,6 @@ class CMakeGeneratorPlugin : public Plugin
         return s;
     }
 
-    void
-    FilterConditionalLibs(QString &links, QString &libs)
-    {
-#ifdef HAVE_VTKM
-        // Will convert vtkm_xxx to vtkm_xxx-version
-        // otherwise will leave it alone.
-        QString vtkmversion = QString("-%1").arg(VTKM_SMALL);
-
-        QStringList newlist = links.split(" ");
-        for(int i = 0; i < newlist.size(); ++i)
-        {
-            QString tmp(newlist[i]);
-            if(tmp.startsWith("vtkm_") && !using_dev)
-            {
```

Comment:
does this file normally have modification comments in it?

Reply:
That's my only feedback for this body of work then. Please add modification comments 🙃

Reply:
did you want the comments for this file to be part of this PR?

Reply:
I think that makes the most sense.